src/libsystemd-network/sd-dhcp6-lease.c

```diff
@@ -147,6 +148,18 @@ DHCP6IA *dhcp6_ia_free(DHCP6IA *ia) {
         return mfree(ia);
 }
 
+sd_dhcp6_option *dhcp6_vendor_option_free(sd_dhcp6_option *option) {
```

Comment:
This is not necessary. We already have free function for the object. Please drop.

Reply:
Fixed